test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -18,6 +19,10 @@ var selectButton = require('../assets/modebar_button');
 describe('Test plot structure', function() {
     'use strict';
 
+    beforeEach(function() {
+        jasmine.addMatchers(customMatchers);
```

Comment:
Is there a way to inject this in the global scope?

That wouldn't have to add this line every time we want to use `toBeCloseToArray`